corehq/apps/users/models.py

```diff
@@ -2302,7 +2302,7 @@ def send_request_email(self):
                                     email_from=settings.DEFAULT_FROM_EMAIL)
 
 
-class Invitation(CachedCouchDocumentMixin, Document):
+class Invitation(Document):
```

Comment:
I think switching to `QuickCachedDocumentMixin` will give all the benefits of caching except the views, which i believe is what we want